core/src/avm2/globals/flash/text/engine/BreakOpportunity.as

```diff
@@ -5,22 +5,22 @@
 
 package flash.text.engine
 {
-    
-    
+
+
     public final class BreakOpportunity
     {
-        // Treats all characters in the ContentElement object as line break opportunities, meaning that a line break will occur afer each character.
+        // Treats all characters in the ContentElement object as line break opportunities, meaning that a line break will occur after each character.
```

Comment:
Actually this typo is canon:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/text/engine/BreakOpportunity.html#ALL

Still I think in a comment, we are free to fix.